src/sfml.mk

```diff
@@ -22,18 +22,22 @@ define $(PKG)_BUILD
     # create pkg-config file
     $(INSTALL) -d '$(PREFIX)/$(TARGET)/lib/pkgconfig'
     (echo 'Name: sfml'; \
-     echo 'Version: 0'; \
+     echo 'Version: $($(PKG)_VERSION)'; \
      echo 'Description: sfml'; \
      echo 'Requires: freetype2 glew openal sndfile vorbisenc vorbisfile'; \
-     echo 'Cflags: -DSFML_STATIC'; \
-     echo 'Libs: -lsfml-audio-s -lsfml-network-s -lsfml-graphics-s -lsfml-window-s -lsfml-system-s'; \
+     $(if $(findstring static,$(TARGET)), \
```

Comment:
Would be better to test $(BUILD_STATIC) and $(BUILD_SHARED). See other .mk files for examples.